pycrostates/utils/_docs.py

```diff
@@ -117,6 +117,18 @@
     Axes on which to draw the colorbar, otherwise the colormap takes
     space from the main axes."""
 
+# ------ Viz -------
+docdict[
+    "picks_fit"
+] = """
+picks : str | list | slice | None
+    Channels to include. Slices and lists of integers will be interpreted
+    as channel indices. Note that all channels must have the same type.
+    In lists, channel name strings (e.g., ['Fp1', 'Fp2']
+    will pick the given channels. "eeg" (default) will pick all eeg channels.
+    Note that channels in info['bads'] will be included if their
+    names or indices are explicitly provided."""
+
```

Comment:
This docstring is not used anywhere else than `_BaseCluster.fit`, so let's move it back to the function docstring directly.